tests/onegov/election_day/views/test_views_api.py

```diff
@@ -0,0 +1,41 @@
+import json
+from datetime import date
+
+import pytest
+from onegov.ballot import Election, ProporzElection
+from onegov.core.utils import module_path
+from onegov.election_day.utils.election.lists import \
+    get_aggregated_list_results
+from tests.onegov.election_day.common import import_wabstic_data
+
+
+# @pytest.mark.parametrize("tar_file", [
```

Comment:
Any reason this is commented out? I'd say either remove it or mark it as pytest.mark.xfail if it is meant to fail.

Reply:
I lack test data at the moment and have to fix the round trip csv import/export that was not working with the data from the test server. It's meant to be un-commented later and tested with a exported test dataset originally fed in by the wabstic cleint of the customer.